src/main/java/org/reactome/release/cosmicupdate/COSMICIdentifierUpdater.java

```diff
@@ -0,0 +1,177 @@
+package org.reactome.release.cosmicupdate;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceDisplayNameGenerator;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.InvalidAttributeValueException;
+import org.reactome.release.common.database.InstanceEditUtils;
+
+/**
+ * Updates a COSMIC Identifier.
+ * Implements <code>Comparable</code> to make sorting easier for reporting purposes. Does *not* override the <code>equals</code> method.
+ * @author sshorser
+ *
+ */
+public class COSMICIdentifierUpdater implements Comparable<COSMICIdentifierUpdater>
+{
+	private static final Logger logger = LogManager.getLogger();
+	private static GKInstance updateWithNewCOSV;
+	private static GKInstance updatePrependCOSM;
+	private String identifier;
+	private Long dbID;
+	private String suggestedPrefix;
+	private boolean valid;
+	private Set<String> mutationIDs = new HashSet<>();
+	private String cosvIdentifier;
+
+	public String getIdentifier()
+	{
+		return identifier;
+	}
+	public void setIdentifier(String identifier)
+	{
+		this.identifier = identifier;
+	}
+	public Long getDbID()
+	{
+		return dbID;
+	}
+	public void setDbID(Long dbID)
+	{
+		this.dbID = dbID;
+	}
+	public String getSuggestedPrefix()
+	{
+		return suggestedPrefix;
+	}
+	public void setSuggestedPrefix(String suggestedPrefix)
+	{
+		this.suggestedPrefix = suggestedPrefix;
+	}
+	public boolean isValid()
+	{
+		return valid;
+	}
+	public void setValid(boolean valid)
+	{
+		this.valid = valid;
+	}
+	public Set<String> getMutationIDs()
+	{
+		return mutationIDs;
+	}
+	public void setMutationIDs(Set<String> mutationIDs)
+	{
+		this.mutationIDs = mutationIDs;
+	}
+	public String getCosvIdentifier()
```

Comment:
What is Cosv? Cosmic Variant?

Reply:
COSV is "Genomic mutation identifier", according to COSMIC.

Reply:
Maybe `getCOSVIdentifier()` since COSMIC uses all capitals?

Reply:
^-- do you think this is worthwhile?